mne/preprocessing/ieeg/_projection.py

```diff
@@ -106,3 +107,92 @@ def project_sensors_onto_brain(info, trans, subject, subjects_dir=None,
         montage_kwargs['ch_pos'][info.ch_names[idx]] = loc
     info.set_montage(make_dig_montage(**montage_kwargs))
     return info
+
+
+@verbose
+def _project_sensors_onto_inflated(info, trans, subject, subjects_dir=None,
+                                   picks=None, max_dist=0.004, flat=False,
+                                   copy=True, verbose=None):
```

Comment:
We should also remove `copy=True` in a follow-up PR and just always operate in-place now that it's private. We can pass `info.copy()` if we want a copy